crates/config/src/transform.rs

```diff
@@ -66,11 +62,34 @@ fn resolve_char(opt: &Option<i32>, dft: i32, len: i32) -> usize {
   }
 }
 
+#[derive(Serialize, Deserialize, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct Replace {
+  source: String,
+  replace: String,
+  by: String,
+  convert: Option<StringConversion>,
```

Comment:
Can we also remove the convert here? Composing transformation can be achieved now by 

```
transform:
  REPLACE:
    replace:
      source: '$A'
      replace: regex
      by: xxx
  CASE:
      convert:
          source: '$REPLACE'
          convert: uppercase
```

Indeed it is more verbose but it is more consistent with other transformations.
And we still have the leeway to add `convert` in future without breaking change.

Reply:
Can do. That also answers my other question – I won't move `source` and `convert` to the "main" Transform level :)